### DISCUSSION TITLE
[Autogen studio] how can i use v0.4 autogen studio?

### DISCUSSION THREAD

**Discussion by soichisumi** [microsoft/autogen#4074] [Q&A] [answered]
Thank you for publicly providing promising tool. I want to use v0.4 of autogen studio. 
Is there any resource to setup v0.4 autogen studio?

I'm referring to the following. can I use this?: https://autogen-studio.com/autogen-studio-ui

**Comment by ekzhu** [comment, marked as answer]
The linked site you referenced is not from us -- we don't manage it. 

AutoGen studio will be updated you can follow the issue page to see the work items

**Reply by soichisumi** [reply]
Thank you for the answer. I'll watch the following and utilize it to estimate timeline
https://github.com/microsoft/autogen/issues?q=is%3Aissue%20state%3Aopen%20label%3Aproj-studio
